### PR TITLE
style: silence two pylint warnings

### PR DESCRIPTION
**Pull request by nedbat**
**Description:**

Describe in a couple of sentences what this PR adds

**JIRA: (Optional)**

[XXX-XXXX](https://openedx.atlassian.net/browse/XXX-XXXX)

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] If adding new checks, followed how_tos/0001-adding-new-checks.rst
- [ ] Changelog record added (if needed)
- [ ] Documentation updated (not only docstrings) (if needed)
- [ ] Commits are squashed

### CODE REVIEW

**Comment by nedbat** [comment]
Go ahead and merge this to get #282 unstuck.

File: edx_lint/pylint/module_trace.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ class ModuleTracingChecker(BaseChecker):
 
     def visit_module(self, node):
         """Called for each module being examined."""
+        self    # pylint: disable=pointless-statement
```
Comment (timmc-edx) [inline edx_lint/pylint/module_trace.py:44]: Why silence rather than removing?

Comment (sarina) [reply edx_lint/pylint/module_trace.py:44]: @nedbat made this commit

Comment (timmc-edx) [reply edx_lint/pylint/module_trace.py:44]: I just found https://github.com/openedx/edx-lint/pull/285 -- which I would have seen earlier, but I though 5.2.5 was the latest release. :-)

**Pull request merged**